e2e/docker-image/docker-image.test.ts

```diff
@@ -0,0 +1,33 @@
+import { createTenv } from '@e2e/tenv';
+
+const { container } = createTenv(__dirname);
+
+beforeAll(async () => {
+  const isCI = !!process.env.CI;
+  const image = isCI ? 'ghcr.io/ardatan/graphql-mesh:test' : 'graphql-mesh';
+
+  container({
+    name: 'graphql-mesh-test',
+    image,
+    containerPort: 4000,
+    hostPort: 4000,
+    healthcheck: [
+      'CMD-SHELL',
+      'echo "this healthcheck is weirdly stuck on checking when http://0.0.0.0:4000 is ready, so this is to bypass it"',
```

Comment:
Should be fixed. A healthcheck is necessary not only for e2e, but also for anyone running Mesh in docker.

Reply:
i got stuck there for sometime, I don't understand why it's not able to execute the healthcheck and return the resolved promise

Reply:
I think it's a crucial thing to have, I agree with @enisdenjo . 
@YassinEldeeb can you please try to get this fixed before we merge?